public/react/components/itemFrames.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "../../style.css";// Imported CSS file
+
+
+const ItemFrame = ({ item }) => {
+  return (
+    <div className="inventory-frame">
+        {/* the entire frame is clickable by wrapping it in a link */}
+        <a href={`?itemId=${item.id}`} className="item-link"> {/* Pass clicked item id (you can see it in the url when clicked) through to the single item view page to display that particular item*/}
```

Comment:
What's the reason for the '?' before itemId?

Reply:
? allows whats in the '' to be added to the URL when clicked
e.g when you click an item on amazon that item appears at the end of the url so you know what item you've clicked
this will help in fetching the data for the single item view